### ISSUE TITLE
Apply universal navigation bar to the problem page

### ISSUE THREAD

**Issue by benetechMatt** [benetech/MathShare#1031]
Move the "save" button and save status message to the right side of the orange bar that has the problem instructions.
Move the "All problems" button to the left side of the orange bar that has the problem instructions.
The orange bar will now contain elements relevant to the problem while the dark blue universal navigation bar will be the same on all pages.
![mockup showing the dark blue universal navigation bar and the orange problem specific bar below it](https://user-images.githubusercontent.com/49413678/76110437-29df5980-5fa4-11ea-8e35-6a800b1884cf.png)
In this mockup the Mathshare logo links back to the dashboard, the personalization settings button is present on the left next to the logo.  On the right is the help menu and the user profile menu button

**Comment by benetechMatt** [comment]
![image](https://user-images.githubusercontent.com/49413678/78187120-7e56e700-7433-11ea-8fe1-236a658cfdf8.png)

Tab order top to bottom, left to right

In the current prompt bar we're adding the current "back to all problems" button, prompt, save button, and saved at time stamp
What is the best HTML element to use for this bar and what is your opinion on what the "skip to main content" should focus?

**Comment by jscholes** [comment]
@benetechMatt The problem needs a main level 1 heading, and the bar of problem-specific controls must appear after it.  So the heading should be the target of the skip link, as on every other page.

In terms of mark-up for the nav bar, it should follow the accessibility advice already set out for the header included on non-problem pages.  It will be best for @rupeshparab to start working on this in a PR which I can then comment on, but it should already start from a good place of accessibility.  It's not a complicated set of controls.

Note: "Sign In" shouldn't be a button, it should be a link.

**Comment by bpmcneilly** [comment]
I agree with @jscholes on the overall structure of this element and placement of the skip link focus. One thing I want to check in on @benetechMatt is that in the first image there appears to be a text to speech button (I'm guessing here, but that's generally what the speaker icon is used for) that is then missing from the second image. If that content is included, that too should be present in the markup after the the H1 element.

**Comment by benetechMatt** [comment]
@alexrcabral the TTS button in the first image was a mistake as it was borrowed for a later version of the MVP. The second image without the speaker icon is correct for this issue.

H1 for the problem
Sign in should be a link as it currently is on the dashboard but keep the current styling

**Comment by alexrcabral** [comment]
Moving to Product Review since PR #1128 was approved and merged

**Comment by bognasoldev89** [comment]
**Tested with success**

Verified on qa env
Chrome v80.0.3987.149

**Verified cases**
- Solving numerous problems (proposed and newly created) by
-- teacher
-- student
-- not signed in user
- Saving problems
- Clicking "All Problems" button
- Checking if the main header works as expected

**Summary**
Navigation bar works fine, the problem is presented correctly, "All problems" button sends to all problems page, "Save" button saves answers and the time of the last save is displayed.
Main header works as expected.

**Attachment**
Navigation bar behavior for not logged in user:
![ms 1031](https://user-images.githubusercontent.com/51324797/82302496-f6fd0d00-99b9-11ea-8a39-75d9dc983e98.gif)
